dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -61,6 +61,10 @@
     @ApiModelProperty(value = "Docker ID of the image", position = 4)
     private String imageID;
 
+    @Column()
+    @ApiModelProperty(value = "Registry image belongs to", position = 5)
+    private String imageRegistry;
```

Comment:
Should this just use an Enum?
See Organization.status for an example. 
The enum would seem to just be io.dockstore.common.Registry